Makefile

```diff
@@ -14,6 +14,7 @@
 #
 # ESPRUINO_1V3=1          # Espruino board rev 1.3 and rev 1v4
 # PICO_1V3=1              # Espruino Pico board rev 1.3
+# PUCKJS=1                # Puck.js board
```

Comment:
This may need an additional directive for the bootloader/dfu as it is possibly best to keep at the top.
Although the doc in the repro may explain it, there are two types of people: those who read the doc and those that read the Makefile (the third type is not in scope)